site/content/device/pc-cooling-fan.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "PC Cooling Fan"
+description = "Move air across to cool a embedded components."
+picture = "/img/pc-cooling-fan.jpg"
++++
+
+# Overview
+[GPIO pins](/device/gpio/) can be leveraged to send pulses and read presses, similar to
+[buzzer](/device/buzzer/) and the [detecting motion](/device/pir/).
+
+Pulses can be sent to a fan to start or stop ventilation.
```

Comment:
This is kinda confusing, instead the fact that the fan's power can be controlled via a PWM is due to the fact that the duty cycle of the signal can be adjusted.